tests/test_functional.py

```diff
@@ -165,15 +182,35 @@ def test_get_conceptscheme_jsonld(self):
         data = json.loads(res.body.decode('utf-8'))
         assert isinstance(data, dict)
         assert 'id' in data
-        assert 'uri'in data
+        assert 'uri' in data
         assert 'label' in data
         assert 'labels' in data
         assert 'sources' in data
-        assert 'notes' in data
-        assert 'type' in data
         assert '@context' in data
         assert '/jsonld/context/skos' in data['@context']
 
+    def test_get_conceptscheme_jsonld_expand(self):
+        
+        res = self.testapp.get(
+            '/conceptschemes/TREES',
+            {},
+            {'Accept': 'application/ld+json'}
+        )
+        assert res.status == '200 OK'
+        assert 'application/ld+json' in res.headers['Content-Type']
+        data = json.loads(res.body.decode('utf-8'))
```

Comment:
In principe is het wel consistent met de rest van de testen, dus ik wil dit wel door de vingers zien en dat er in de toekomst eens een grotere cleanup gebeurt.

Maar de propere maniervoor deze blok had deze geweest:

```py
res = self.testapp.get(
    '/conceptschemes/TREES',
    headers={'Accept': 'application/ld+json'},
    status=200,  # er wordt sowieso getest op success state (200-299), maar als je specifiek voor 200 wilt testen kan dit...
)
assert 'application/ld+json' in res.headers['Content-Type']
data = res.json
```